public/about/index.html

```diff
@@ -1,26 +1,26 @@
 <!DOCTYPE html>
 <html lang="en-us">
-  <head>
+  <head><script src="/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=livereload" data-no-instant defer></script>
   <meta charset="utf-8">
   <title>About Us</title>
 
-  <meta name="generator" content="Hugo 0.79.1" />
+  <meta name="generator" content="Hugo 0.128.1">
 
   <!-- mobile responsive meta -->
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
   
   <!-- ** Plugins Needed for the Project ** -->
   <!-- Bootstrap -->
-  <link rel="stylesheet" href="https://pysal.org/plugins/bootstrap/bootstrap.min.css">
+  <link rel="stylesheet" href="http://localhost:1313/plugins/bootstrap/bootstrap.min.css">
```

Comment:
looks like this has accidentally swapped in the preview URLs?

Reply:
Ah, my bad; the public files should not have been committed here.
I think they will be rebuilt (correctly)  once the action runs.